src/ucp/api/ucp.h

```diff
@@ -1096,6 +1187,29 @@ unsigned ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Poll stream oriented events on a specific worker.
+ *
+ * This non-blocking routine returns number of endpoints where requested events
```

Comment:
`This non-blocking routine returns endpoints on a worker which have ready streaming data. The ready endpoints are placed in @poll_eps array, and the function return value indicates how many are there.`